tethysext/atcore/services/workflow_manager/condor_workflow_manager.py

```diff
@@ -310,3 +314,15 @@ def run_job(self):
         # Execute
         self.workflow.execute()
         return str(self.workflow.id)
+
+    def validate_jobs(self, jobs):
+        """
+        Validates that the jobs are defined (not empty) and are a CondorWorkflowJobNode or equivalent dicaiontry.
+
+        Args:
+            jobs(list<CondorWorkflowJobNode or dict>): List of CondorWorkflowJobNodes to run.
+        """
+        if not inspect.isfunction(jobs):
+            if not jobs or not all(isinstance(x, (dict, CondorWorkflowJobNode)) for x in jobs):
+                raise ValueError('Argument "jobs" is not defined or empty. Must provide at least one '
```

Comment:
Change Argument -> Given
This won't always be raised in the constructor, so "Argument" may cause confusion.

Reply:
Changed